apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -320,7 +321,14 @@ public void captureException(long epochMicros, @Nullable Throwable e, TraceConte
         captureException(epochMicros, e, parent, null);
     }
 
-    public void captureException(long epochMicros, @Nullable Throwable e, @Nullable TraceContextHolder<?> parent, @Nullable ClassLoader initiatingClassLoader) {
+    @Nullable
+    public String captureExceptionAndGetErrorId(long epochMicros, @Nullable Throwable e, @Nullable TraceContextHolder<?> parent) {
```

Comment:
I don't think we need this API, it doesn't save much to the caller (only extract the ID). The one returning the `ErrorCapture` is very useful as is. 
I can easily imaging how a caller may want to do lots of other stuff with the `ErrorCapture` and we wouldn't add an API for each usage.

Reply:
fixed

Reply:
added documentation